yarn/tests/test_unit.py

```diff
@@ -13,6 +13,11 @@
     'tags, expected_tags',
     [
         pytest.param("test:example", ["app_test:example"], id='tag_key_value'),
+        pytest.param("test:example,,test", ["app_test:example", "app_", "app_test"], id='test_empty_tag'),
+        pytest.param("test:example,:,test", ["app_test:example", "app_:", "app_test"], id='test_empty_kv_tag'),
+        pytest.param(
+            "test:example,test::test", ["app_test:example", "job_tag:test:example,test::test"], id='test_failure'
```

Comment:
Should we add a test to `test_yarn.py` to test that these malformed tags fail properly and don't get sent?

Reply:
These tags do get sent no matter what: https://github.com/DataDog/integrations-core/blob/master/yarn/datadog_checks/yarn/yarn.py#L284